src/Dashboard/Http/Controllers/AuthenticateDashboard.php

```diff
@@ -2,13 +2,29 @@
 
 namespace BeyondCode\LaravelWebSockets\Dashboard\Http\Controllers;
 
+use Pusher\Pusher;
 use Illuminate\Http\Request;
-use Illuminate\Contracts\Broadcasting\Broadcaster;
+use BeyondCode\LaravelWebSockets\Apps\App;
+use Illuminate\Broadcasting\Broadcasters\PusherBroadcaster;
 
 class AuthenticateDashboard
 {
-    public function __invoke(Request $request, Broadcaster $broadcaster)
+    public function __invoke(Request $request)
     {
+        /**
+         * Find the app by using the header
+         * and then reconstruct the PusherBroadcaster
+         * using our own app selection.
+         */
+        $app = App::findById($request->header('x-app-id'));
+
+        $broadcaster = new PusherBroadcaster(new Pusher(
+            $app->key,
+            $app->secret,
+            $app->id,
+            $config['options'] ?? []
```

Comment:
Can you remove the `$config['options']` part?
Since the $config variable will never be available, we don't need it.

Reply:
eh.. alright. it didn't throw any error though. Was it because of the ?? part?